packages/manager/src/index.ts

```diff
@@ -13,6 +13,8 @@ export { createSliceMachineManager } from "./managers/createSliceMachineManager"
 export { createSliceMachineManagerMiddleware } from "./managers/createSliceMachineManagerMiddleware";
 export type { CreateSliceMachineManagerMiddlewareArgs } from "./managers/createSliceMachineManagerMiddleware";
 
+export type { GitRepoSpecifier, Owner } from "./managers/git/types";
```

Comment:
Could we rename `Owner` to `GitOwner`? A top-level export named `Owner` has ambiguous meaning.

Reply:
The rename can happen throughout the code, not just as an export rename.